packages/otp2-tile-overlay/src/Otp2TileOverlay.story.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import BaseMap from "@opentripplanner/base-map";
+import generateOTP2TileLayers from ".";
+
+export default {
+  title: "OTP2 Tile Layer"
+};
+
+export const OtpTileLayer = (): JSX.Element => {
+  const [endpoint, setEndpoint] = useState("");
+  return (
+    <>
+      {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+      <label>
+        OTP2 Server with stops layer enabled
```

Comment:
Instead of/in addition to using a server, I recommend including a mock of a simple tile layer with a few entities, as we already do with the overlay packages to illustrate the inputs.

Reply:
This was much more work than expected but we now mock tile server data using `msw`